data_extraction/notebooks/retrieval_from_pdf_tables.ipynb

```diff
@@ -552,7 +552,7 @@
     "load_dotenv(os.path.join(repo_dir,'.env'))\n",
     "\n",
     "llm = SambaNovaEndpoint(\n",
-    "    model_kwargs={\"do_sample\": False, \"temperature\": 0.0},\n",
+    "    model_kwargs={\"do_sample\": False, \"temperature\": 0.0, \"select_expert\": \"llama-2-7b-chat-hf\"},\n",
```

Comment:
here please add a comment that indicates select expert should be removed if not using CoE